src/main/java/fatec/labdev/projeto/pollingapp/option/service/OptionServiceImpl.java

```diff
@@ -35,4 +40,25 @@ public Option save(Option option) {
     public void deleteById(UUID id) {
         this.optionRepository.deleteById(id);
     }
+
+    @Override
+    @Transactional
```

Comment:
Usar o @Transactional no nivel da classe

Reply:
Eu não sei a diferença. Tem referência aí?

Reply:
Aqui no caso é apenas pra manter um padrão, geralmente é usado no nível da classe para evitar futuros erros quando novos métodos entrarem.

Method x Class: https://stackoverflow.com/questions/23132822/what-is-the-difference-between-defining-transactional-on-class-vs-method#:~:text=The%20%40Transactional%20annotation%20on%20the,Transaction%20management%20in%20Spring